.gitignore

```diff
@@ -7,4 +7,5 @@ jupyterhub/data-science-notebook/requirements.txt
 jupyterhub/data-science-notebook/spark-defaults.conf
 jupyterhub/package-dependencies/requirements.txt
 jupyterhub/pyspark-notebook/requirements.txt
-airflow/requirements.txt
\ No newline at end of file
+airflow/requirements.txt
+airflow/current/requirements.txt
```

Comment:
Why do we have two `requirements.txt` for airflow? Are both needed?

Reply:
they are created on the fly and I just added them to gitignore because I am annoyed by uncommited files. line 10 is probably from old airflow and line 11 I added for new airflow. It shows a change for line 1o probably because of the added line break